ocs_ci/ocs/cluster.py

```diff
@@ -1304,11 +1304,11 @@ def check_osd_tree_3az_aws(osd_tree, number_of_osds):
     return check_osds_in_hosts_osd_tree(all_hosts_flatten, osd_tree)
 
 
-def check_osd_tree_1az_aws(osd_tree, number_of_osds):
+def check_osd_tree_1az(osd_tree, number_of_osds):
```

Comment:
Same as my previous comment - The name `check_osd_tree_1az_cloud` will be better here.

Reply:
done